### PR TITLE
Jd 2022 08 fix header status bar

### LINKED ISSUE #350: back button on ios iphone SE
back button has too little top padding on ios device so half overlaps with status stuff

### PR DESCRIPTION
**Pull request by goapunk**
closes #350

### CODE REVIEW

File: containers/Ideas/IdeaProject.js [hunk 1]
```diff
@@ -87,11 +87,11 @@ export const IdeaProject = (props) => {
         source={{ uri: bgImage }}
         accessibilityIgnoresInvertColors={true}
       />
+      <LinearGradient
```
Comment (goapunk) [inline containers/Ideas/IdeaProject.js:90]: I'm not super sure about this but it really looks weird if we don't move it here. Or maybe we need to have a negative margin on the scrollview or something

File: containers/Ideas/IdeaCreate.styles.js [hunk 2]
```diff
@@ -7,7 +7,9 @@ import { SPACINGS } from '../../theme/spacings'
 export const styles = StyleSheet.create({
   container: {
     paddingHorizontal: SPACINGS.multiplyBy(0.75),
-    backgroundColor: COLORS.paper
+  },
+  header: {
+    paddingTop: SPACINGS.multiplyBy(0.25)
```
Comment (phillimorland) [inline containers/Ideas/IdeaCreate.styles.js:12]: I wonder if this should just be on the header and we unify the top padding on all containers so we don't have it set individually all over the place?

File: containers/Reports/ReportCreateMessage.js [hunk 3]
```diff
@@ -46,8 +47,10 @@ export const ReportCreateMessage = props => {
   }
 
   return (
-    <View style={styles.container}>
-      <Header navigation={props.navigation} />
+    <SafeAreaView style={styles.container}>
+      <View style={styles.header}>
```
Comment (phillimorland) [inline containers/Reports/ReportCreateMessage.js:51]: same as above about unifying padding

**Review by phillimorland — APPROVED** [review]
works like a charm, just question about trying to unify padding but I don't feel it's important enough not to merge but we should look into it

**Pull request merged**